Makefile

```diff
@@ -4,8 +4,8 @@
 # Fast quality checks (for pre-commit)
 quality-fast:
 	@echo "Running fast quality checks..."
-	black --check cal/ sample/
-	ruff check cal/ sample/
+	black --check cal/ sample/ util/
+	ruff check cal/ sample/ util/
```

Comment:
Added util here since the decorator code in there that I'd added needs to be checked too.